test/cql-pytest/test_keyspace.py

```diff
@@ -99,7 +99,7 @@ def test_drop_keyspace_nonexistent(cql):
 # Test trying to ALTER a keyspace.
 def test_alter_keyspace(cql, this_dc):
     with new_test_keyspace(cql, "WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', '" + this_dc + "' : 1 }") as keyspace:
-        cql.execute(f"ALTER KEYSPACE {keyspace} WITH REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 3 }} AND DURABLE_WRITES = false")
+        cql.execute(f"ALTER KEYSPACE {keyspace} WITH REPLICATION = {{ 'class' : 'NetworkTopologyStrategy', '{this_dc}' : 2 }} AND DURABLE_WRITES = false")
```

Comment:
we should test that we get the expected error when attempting to change RF by more than 1,
and that nothing explodes, and the keyspace is usable after that, including a correct ALTER

Reply:
Added a similar testcase that changes RF from 1 to 3 and expects `InvalidRequest` to be raised.